#!/usr/bin/env python
PACKAGE = "corobot_camera"
# The following line should not be here for Groovy and above
import roslib;roslib.load_manifest(PACKAGE)

# For Groovy and above, the following line should be
# from dynamic_reconfigure.parameter_generator_catkin import *
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("camera_activated", bool_t, 0, "Activate the camera or not", True)


exit(gen.generate(PACKAGE, "corobot_camera", "corobot_camera"))
